cluster-controller/src/main/java/io/strimzi/controller/cluster/resources/AbstractCluster.java

```diff
@@ -482,6 +486,7 @@ protected Deployment createDeployment(
                 .withAnnotations(podAnnotations)
                 .endMetadata()
                 .withNewSpec()
+                .withServiceAccountName(getServiceAccountName())
```

Comment:
How will it affect other deployments when you set their SA to null? Because I guess they use by default the default SA which is in every namespace.

Reply:
Good point. The only "cluster" to use that is the Kafka Connect one. I have to test it.

Reply:
I have just tried deploying the Kafka Connect cluster and it seems to start and work without problems/errors. Setting `null` seems to revert to the `default` service account.